### PR TITLE
Feature/public task

### PR DESCRIPTION
**Pull request by huibvanstraten**
documentation for the new Public Task plugin

### CODE REVIEW

File: using-valtimo/plugin/public-task/configure-public-task-plugin.md
Comment (Klaas-Ritense) [inline using-valtimo/plugin/public-task/configure-public-task-plugin.md:1]: This file should be linked in the SUMMARY.md

File: using-valtimo/plugin/public-task/configure-public-task-plugin.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Public Task Plugin
+
+The Public Task Plugin exposes a User Task outside of the Valtimo user environment. It generates an HTML, an assignee candidate receives a URL and is able to submit the task within the HTML. 
+From there Valtimo will pick up the submission and complete the task.
+
+The Public Task Plugin is not a standard part of Valtimo. It is available in the Plugins Repo. More information can be found [here](https://docs.valtimo.nl/readme/modules/plugin-introduction#plugins-repo).
```
Comment (Klaas-Ritense) [inline using-valtimo/plugin/public-task/configure-public-task-plugin.md:6]:
```suggestion
The Public Task Plugin is not a standard part of Valtimo. It is available in the Plugins Repo. More information can be found [here](/introduction/modules/plugin-introduction.md#plugins-repo).
```

File: using-valtimo/plugin/public-task/img/configure-plugin.png
Comment (Klaas-Ritense) [inline using-valtimo/plugin/public-task/img/configure-plugin.png:1]: Looks like the 'configurationTitle' is still missing in the translation file of the plugin

File: using-valtimo/plugin/public-task/configure-public-task-plugin.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Public Task Plugin
```
Comment (marijnritense) [inline using-valtimo/plugin/public-task/configure-public-task-plugin.md:1]: Formatting is messed up around the images. Please make use of newlines in order to break everything up correctly.

**Comment by marijnritense** [comment]
This documentation appears to be for a non-valtimo plugin. Please move this over to your other repository.

**Comment by huibvanstraten** [comment]
@marijnritense It is a "Plugins Repo" plugin, build in general for any Valtimo implementation.  I would say in the same category as smtp-mail, which is also in here.

**Comment by marijnritense** [comment]
I think that should be removed as well then. If it's not a part of Valtimo, it should not be in the official valtimo documentation.

**Comment by marijnritense** [comment]
After discussing with @ThomasMinkeRitense I am closing this PR, as we do not maintain documentation for code outside of our own repositories.

**Pull request closed without merging**